docs/sources/mimir/release-notes/v2.11.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Grafana Mimir version 2.11 release notes"
+menuTitle: "V2.11 release notes"
+description: "Release notes for Grafana Mimir version 2.11"
+weight: 1000
+---
+
+# Grafana Mimir version 2.11 release notes
+
+Grafana Labs is excited to announce version 2.11 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
```

Comment:
I'm biased of course, but I think ingester to querier chunks streaming and ingester query request minimisation are both worth mentioning here, as they're on by default for the first time in this release.

Chunks streaming reduces the memory utilisation of queriers, reducing the likelihood of OOMing, while ingester query request minimisation reduces the number of query requests to ingesters, improving performance and resource utilisation for both queriers and ingesters.